### ISSUE TITLE
Config loader quietly falls back to defaults when it encounters an error

### ISSUE THREAD

**Issue by good-idea** [pedronauck/docz#69]
## Bug Report

**Describe the bug**

When `doczrc.js` or anything imported within it contains an error, it stops loading custom config and falls back to the defaults. 

*A clear and concise description of what the bug is.*

**To Reproduce**
in your `doczrc.js`:

```
console.log('Works')
const a = 1
a = 2

console.log('this will not print')

module.exports = {
        source: './src',
        debug: true,
}
```

**Expected behavior**

The command should throw the error and exit. Instead, docz continues to run, (presumably) using the default config.

**Comment by good-idea** [comment]
Made a PR!

**Comment by pedronauck** [comment]
## 🎉&nbsp; Fixed and released in the new [v0.4.0](https://github.com/pedronauck/docz/releases/tag/v0.4.0)
